autoapi/mappers/python/objects.py

```diff
@@ -100,6 +100,22 @@ def display(self):
 
         :type: bool
         """
+        try:
+            skip = (
+                self.app.emit_firstresult(
+                    "autoapi-skip-member",
+                    self._class_content,
```

Comment:
Should this be `self.type`? `self._class_content` stores the value of the `autoapi_python_class_content` option rather than the type of object. You might need to make sure that `type` is correct for each type of object.

Reply:
Ok, i will write appropriate tests